gateway.go

```diff
@@ -23,6 +23,9 @@ var orderedResources = []*resourceWithIndex{
 	{
 		name: "HTTPRoute",
 	},
+	{
+		name: "VirtualServer",
```

Comment:
can you also update this with this little change https://github.com/ori-edge/k8s_gateway/pull/72

Reply:
Done